### DISCUSSION TITLE
feature request: Add recice to add cinnamon.nvim

### DISCUSSION THREAD

**Discussion by hacker-DOM** [LazyVim/LazyVim#724] [Q&A]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

I would like to use smooth scrolling with Cinnamon.nvim

Per their docs I have placed `vim.keymap.set('n', 'gd', "<Cmd>lua Scroll('definition')<CR>")` in my `lua/config/keymaps.lua` file, but `gd` still doesn't smooth scroll. I suspect it is because the lsp plugin's on_attach function overrides my keymap.

### Describe the solution you'd like

Since a lot of beginners are likely to run into this problem, it would be great to add a recipe for it

### Describe alternatives you've considered

I'm considering trying to override `lazyvim.plugins.lsp.keymaps.get()` function..

### Additional context

N/a

**Comment by dpetka2001** [comment]
You can change Lsp keymaps per the [docs](https://www.lazyvim.org/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps). Try it out.
